resources/js/Components/ComponentsPraktikans/PraktikanAnno.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import notificationIcon from "../../../assets/Icon-Notification.svg";
+import CloseButton from "../CloseButton";
+
+export default function PraktikanAnnouncement() {
+    // State to manage popup visibility
+    const [isPopupOpen, setIsPopupOpen] = useState(false);
+    const isHaveAnnounce = true;
+
+    // Function to open the popup
+    const openPopup = () => {
+        setIsPopupOpen(true);
+    };
+
+    // Function to close the popup
+    const closePopup = () => {
+        setIsPopupOpen(false);
+    };
+
+    // dummy message
+    let message =
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta. Sed nec diam eu diam mattis viverra. Nulla fringilla, orci ac euismod semper, magna diam porttitor mauris, quis sollicitudin sapien justo in libero. Fusce varius, ligula non tempus aliquam, nunc turpis ullamcorper nibh, in tempus sapien eros vitae ligula. Aliquam erat volutpat. Ut adipiscing porttitor metus, a malesuada neque ultricies vel. Duis consequat vehicula dui, sit amet tincidunt lorem tempor ut. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta. Sed nec diam eu diam mattis viverra. Nulla fringilla, orci ac euismod semper, magna diam porttitor mauris, quis sollicitudin sapien justo in libero. Fusce varius, ligula non tempus aliquam, nunc turpis ullamcorper nibh, in tempus sapien eros vitae ligula. Aliquam erat volutpat. Ut adipiscing porttitor metus, a malesuada neque ultricies vel. Duis consequat vehicula dui, sit amet tincidunt lorem tempor ut.";
+
+    return (
+        <>
+            <div className="absolute right-0 top-0">
+                <button onClick={openPopup} className="mr-10 mt-3">
+                    <img
+                        className="w-12"
+                        src={notificationIcon}
+                        alt="notification"
+                    />
+                </button>
+            </div>
+            {isPopupOpen && isHaveAnnounce && (
+                <div
+                    id="upEdit"
+                    className="absolute z-40 h-screen w-screen bg-black bg-opacity-60 flex justify-center items-center"
+                >
+                    <div className="bg-white w-[440px] h-[550px] rounded-md flex flex-col">
+                        <div className="w-full text-right">
+                            <CloseButton closePopup={closePopup} />
+                        </div>
+                        <div className="font-poppins flex flex-col flex-grow p-4">
+                            <h2 className="text-[#D61B00] text-center text-2xl font-bold">
+                                Pengumuman
```

Comment:
You can also just change the text according to the value of `isAnnouncementExist` here

Reply:
I will change it ASAP